core/themes/seven/css/style.css

```
@@ -890,7 +890,14 @@ div.teaser-checkbox .form-item,
 .form-item input.form-radio,
 .form-item input.form-checkbox {
   vertical-align: middle;
-  margin-right: 5px;
+}
+[dir="ltr"] .form-item input.form-radio,
+[dir="ltr"] .form-item input.form-checkbox {
```

Although the result is the same, we don't usually specify `[dir="ltr"]` we should remove this prefix for consistency.

```suggestion
.form-item input.form-radio,
.form-item input.form-checkbox {
```